project.clj

```diff
@@ -6,19 +6,20 @@
 
   :source-paths ["src/cljs"]
 
-  :dependencies [[org.clojure/clojure "1.7.0"]
+  :dependencies [[org.clojure/clojure "1.8.0"]
```

Comment:
Does this have a hard dependency on 1.8?  I don't think all of our other apps are updated to 1.8 yet.